db/migrate/20131030184435_create_houses.rb

```diff
@@ -1,10 +1,10 @@
 class CreateHouses < ActiveRecord::Migration
   def change
     create_table :houses do |t|
-      f.string :name, null: false
-      f.string :img_url
+      t.string :name, null: :false
```

Comment:
You don't need a colon before `false`. It's a boolean, not a symbol. You do still need it after `null` though.